xml/soc-operations-networking.xml

```diff
@@ -0,0 +1,166 @@
+       A reboot will be required for <literal>iommu_groups</literal> to be
+       enabled.
+      </para>
+     </step>
+     <step>
+      <para>
+       After the reboot, check that IOMMU is enabled:
+      </para>
+<screen>&prompt.root;virt-host-validate
+.....
+QEMU: Checking if IOMMU is enabled by kernel
+: PASS
+.....</screen>
+     </step>
+     <step>
+      <para>
+       Confirm IOMMU groups are available by finding the group associated with
+       your PCI device (for example Nvidia GPU):
+      </para>
+<screen>&prompt.ardana;lspci -nn | grep -i nvidia
+08:00.0 VGA compatible controller [0300]: NVIDIA Corporation GT218 [NVS 300] [10de:10d8] (rev a2)
+08:00.1 Audio device [0403]: NVIDIA Corporation High Definition Audio Controller [10de:0be3] (rev a1)</screen>
+      <para>
+       In this example, <literal>08:00.0</literal> and
+       <literal>08:00.1</literal> are addresses of the PCI device. The vendorID
+       is <literal>10de</literal>. The productIDs are <literal>10d8</literal>
+       and <literal>0be3</literal>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Confirm that the devices are available for passthrough:
+      </para>
+<screen>&prompt.ardana;ls -ld /sys/kernel/iommu_groups/*/devices/*08:00.?/
+drwxr-xr-x 3 root root 0 Feb 14 13:05 /sys/kernel/iommu_groups/20/devices/0000:08:00.0/
+drwxr-xr-x 3 root root 0 Feb 19 16:09 /sys/kernel/iommu_groups/20/devices/0000:08:00.1/</screen>
+      <note>
+       <para>
+        With PCI passthrough, only an entire IOMMU group can be passed. Parts
+        of the group cannot be passed. In this example, the IOMMU group is
+        <literal>20</literal>.
+       </para>
+      </note>
+     </step>
+    </procedure>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis role="bold">Preparing &o_comp; and &o_img; for
+     passthrough</emphasis>
+    </para>
+    <procedure>
+      <step>
+        <para>
+          Create a new file to host configurations mentioned in
+          <link xlink:href="https://docs.openstack.org/nova/pike/admin/pci-passthrough.html"/>.
+          Both <literal>nova-compute</literal> and <literal>nova-scheduler</literal>
+          must be configured.
+        </para>
+        <note>
+          <para>
+            If you add them on the existing <literal>nova.conf</literal>, then it
+            is overwritten by the <literal>chef-client.service</literal> and
+            restored to the original version of <literal>nova.conf</literal>.
+          </para>
+        </note>
+      </step>
+      <step>
+        <para>
+          Add the following configuration to <literal>nova.conf</literal> on
+          the controller node:
+        </para>
+        <screen>
+          /etc/nova/nova.conf.d # ls
+          010-nova.conf  100-nova.conf  101-nova-placement.conf
+        </screen>
+      </step>
+      <step>
+        <para>
+          Add filter <literal>PciPassthroughFilter</literal> to <literal>enabled_filters</literal>.
```

Comment:
As for this paragraph for adding PciPassthroughFilter, we propose a [PR](https://github.com/crowbar/crowbar-openstack/pull/2247) to set it as a default value when user choose all filters. I think we can ignore this paragraph for now.